sellotape/sellotape_dj/sellotape_dj/settings.py

```diff
@@ -118,3 +128,38 @@
 # https://docs.djangoproject.com/en/3.1/howto/static-files/
 
 STATIC_URL = '/static/'
+
+LOGIN_URL = 'login'
+LOGIN_REDIRECT_URL = 'home'
+LOGOUT_URL = 'logout'
+LOGOUT_REDIRECT_URL = 'login'
+
+SOCIAL_AUTH_PIPELINE = (
+    'social_core.pipeline.social_auth.social_details',
+    'social_core.pipeline.social_auth.social_uid',
+    'social_core.pipeline.social_auth.social_user',
+    'social_core.pipeline.user.get_username',
+    'social_core.pipeline.social_auth.associate_by_email',
+    'social_core.pipeline.user.create_user',
+    'social_core.pipeline.social_auth.associate_user',
+    'social_core.pipeline.social_auth.load_extra_data',
+    'social_core.pipeline.user.user_details',
+)
+
+SOCIAL_AUTH_GOOGLE_OAUTH2_KEY = '87810693431-99al9o78efneb8dpuvhkkldmdon2n7tm.apps.googleusercontent.com'
```

Comment:
are you sure you want to leak your key to the entire internet?

Reply:
I didn't think it will matter as we're in development stages, and this is only for localhost. But of course, as a best practice it should be avoided. I'll remove this and send them in the Slack group chat for better security.